src/components/FullDetailView/AbstractFullDetailsView.js

```diff
@@ -33,7 +35,8 @@ export class AbstractFullDetailsView extends Component {
       <Modal
         closable={false}
         footer={null}
-        centered
+        // if it's mobile do not center the Modal
+        centered={!isMobile}
```

Comment:
I'm seeing the modal open halfway down the page on mobile width.
Is there anything we can do to align it to the top?